cvmfs/mountpoint.cc

```diff
@@ -1831,7 +1831,41 @@ bool MountPoint::SetupBehavior() {
                optarg.c_str());
     }
   }
-  magic_xattr_mgr_ = new MagicXattrManager(this, xattr_visibility);
+
+  std::set<gid_t> protected_xattr_gids;
+  if (options_mgr_->GetValue("CVMFS_XATTR_PRIVILEGED_GIDS", &optarg)) {
+    std::vector<string> tmp = SplitString(optarg, ',');
+
+    for (size_t i = 0; i < tmp.size(); i++) {
+      LogCvmfs(kLogCvmfs, kLogSyslogWarn | kLogDebug,
```

Comment:
We show the full list when we do the sanity check
```suggestion
      LogCvmfs(kLogCvmfs, kLogDebug,
```